plugins/pulp_rpm/plugins/distributors/rsync/distributor.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2013 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the License
+# (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied, including the
+# implied warranties of MERCHANTABILITY, NON-INFRINGEMENT, or FITNESS FOR A
+# PARTICULAR PURPOSE.
+# You should have received a copy of GPLv2 along with this software;
+# if not, see http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt
+
+from pulp.common.config import read_json_config
+from pulp.plugins.distributor import Distributor
+from pulp.plugins.rsync import configuration
+from pulp_rpm.plugins.distributors.rsync import publish
+import logging
+
+from pulp_rpm.common.ids import (TYPE_ID_DISTRO, TYPE_ID_DRPM, TYPE_ID_ERRATA, TYPE_ID_PKG_GROUP,
+                                 TYPE_ID_PKG_CATEGORY, TYPE_ID_RPM, TYPE_ID_SRPM,
+                                 TYPE_ID_YUM_REPO_METADATA_FILE)
+
+_LOG = logging.getLogger(__name__)
+
+TYPE_ID_DISTRIBUTOR_RPM_RSYNC = 'rpm_rsync_distributor'
+CONF_FILE_PATH = 'server/plugins.conf.d/%s.json' % TYPE_ID_DISTRIBUTOR_RPM_RSYNC
```

Comment:
That directory is full of config files for users to change settings in plugins. Is it that this plugin doesn't have any settings that should be changed globally? Or is there some other reason?